test/meson.build

```diff
@@ -0,0 +1,5 @@
+executable(
+    'cmenu-test',
+    'main.c',
+    dependencies: cmenu_dep,
+)
```

Comment:
How does this test binary get run? Could you define it as a meson test, and then run it in CI and distro packaging?

Reply:
We could probably run it through meson, and it might not be a bad idea to add in the extra verification step in the build process. But my original reason for adding this was to make it easier to test and diagnose things like segfaults without having to actually install. (I kept forgetting to reinstall the working version and then shutting down my comp, so that Cinnamon would crash immediately upon starting back up so I'd have to reinstall an older version from fallback and then restart Cinnamon - it got very annoying.) Plus it's easier to use gdb this way too.